l10n_de_tax_statement_zm/tests/test_l10n_de_tax_statement_zm.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2018 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import UserError, ValidationError
+
+from odoo.addons.l10n_de_tax_statement.tests.test_l10n_de_tax_statement\
+    import TestVatStatement
+
+
+class TestTaxStatementZM(TestVatStatement):
+
+    def _prepare_zm_invoice(self):
+        for invoice_line in self.invoice_1.invoice_line_ids:
+            for tax_line in invoice_line.invoice_line_tax_ids:
+                tax_line.tag_ids = self.tag_41_base
```

Comment:
```suggestion
                tax_line.tag_ids = self.tag_3
```

Reply:
I have applied your proposed change. Thanks a lot for your intervention!